src/itest/java/uk/gov/companieshouse/insolvency/data/repository/RepositoryITest.java

```diff
@@ -9,6 +9,7 @@
 import org.springframework.boot.autoconfigure.mongo.embedded.EmbeddedMongoAutoConfiguration;
 import org.springframework.boot.test.autoconfigure.data.mongo.DataMongoTest;
 import org.testcontainers.junit.jupiter.Testcontainers;
+import uk.gov.companieshouse.GenerateEtagUtil;
 import uk.gov.companieshouse.api.insolvency.CompanyInsolvency;
```

Comment:
I dont see a Repository test that persists and fetches to see json to bson is successfully converted, if not exists please add. Refer to charge-data-api component which has similar tests

Reply:
The tests have been covered as part of DSND-503